joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller.hpp

```diff
@@ -90,6 +95,8 @@ class JointTrajectoryController : public controller_interface::ControllerInterfa
   rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
   on_shutdown(const rclcpp_lifecycle::State & previous_state) override;
 
+  rclcpp::Duration get_action_monitor_period() {return action_monitor_period_;}
```

Comment:
what is an `action_monitor` and why does it have to be public API?

Reply:
removed e68ead9
action_monitor is the rate at which the nonRealtime loop runs